frontend/src/components/Collections/components/Collection/common/components/DeleteCollection/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, H6, Intent } from "@blueprintjs/core";
```

Comment:
ah I'm thinking `/DeleteCollection` can just live at `src/components/Collections/components/DeleteCollection` to prevent too much nesting. What do you think? 

Thank you!

Reply:
That works with me!